### DISCUSSION TITLE
Uninstalled cask remains in list

### DISCUSSION THREAD

**Discussion by dunkelmunkel** [Homebrew/discussions#2748] [Casks] [answered]
Hello everyone,

I uninstalled the cask gimp, but it remains in the list of casks when I call 

`brew list`

Installiing and uninstalling the cask gimp cannot does not solve the problem.
How can I correct the incorrect list?

Christian

**Comment by SMillerDev** [comment, marked as answer]
How did you uninstall it?

**Reply by dunkelmunkel** [reply]
I uninstalled it with

`brew uninstall gimp`

**Reply by neutric** [reply]
Please open an [issue](https://github.com/Homebrew/homebrew-cask/issues). The information provided through the bug template will help analyse what might be going wrong.

**Reply by dunkelmunkel** [reply]
ok, I will do that. Thank you!

**Reply by neutric** [reply]
Sure thing! Please mark as answered so that other contributors don't spend their time opening and reading this for nothing.

**Reply by dunkelmunkel** [reply]
ok! :)
